test/dlc_tests/container_tests/bin/test_mx_dlc_telemetry_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import numpy as np
+import time
+
+def opt_in_opt_out_test():
+  os.environ['TEST_MODE']='1'
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="True"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==False)
+
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="False"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==True)
+
+
```

Comment:
nit: Inconsistent spacing - why is there a 2 line space between these two blocks?

Reply:
Addressed.